node/src/components/storage.rs

```diff
@@ -1452,10 +1539,6 @@ impl Storage {
     }
 
     /// Reads all known deploy hashes from the internal store.
-    ///
```

Comment:
Shouldn't this comment block remain unchanged?  It's still valid and contains useful info.  Or did you mean to update this method to return a `Result` too?

Reply:
My bad.  I've reverted this in ce02f1da6